src/generic/Matrix.jl

```diff
@@ -6019,7 +6024,7 @@ end
 ################################################################################
 
 @doc Markdown.doc"""
-    zero_matrix(R::Ring, r::Int, c::Int) -> MatElem
+    zero_matrix(R::Ring, r::Int, c::Int)
```

Comment:
I personally do not like adding return types, so I'm ok with this change. However others in our group disagree and want them to be included sometimes and don't want them to be removed once added.

Not my decision, but one we have to live with.

Reply:
Thanks for your review, I will change this back!